pd/main.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 	"os"
 
+	. "github.com/PagerDuty/go-pagerduty/command"
```

Comment:
I forgot to include this in the review comment. Please don't use dot imports, except for this one specific case:

https://github.com/golang/go/wiki/CodeReviewComments#import-dot